core/tests/protractor_utils/CommunityDashboardTranslateTextTab.js

```diff
@@ -26,8 +26,12 @@ var CommunityDashboardTranslateTextTab = function() {
     by.css('option:checked'));
 
   var _selectLanguage = async function(language) {
+    await selectableLanguageElements.click();
```

Comment:
Why do we need this? I think usually you can click directly on dropdown elements without clicking on the parent. I think your way is fine; I'm just trying to understand why it's different

Reply:
The previous element was simply a <select /> component. This selector is a custom component, and the dropdown is set to display:none until it is clicked. Without the click, the next step would fail

Reply:
makes sense!